demos/more_examples/graphistry_features/layout_map.ipynb

```diff
@@ -0,0 +1,616 @@
+    "\n",
+    "For demonstration purposes, we'll create synthetic relationships between companies with timestamps and relationship types."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d8fcb58",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "# Create pairs of companies. For simplicity, let's create random pairs.\n",
+    "# In a real scenario, you would likely have a specific way to determine relationships.\n",
+    "num_edges = 20000  # You can adjust the number of edges\n",
+    "src_companies = np.random.choice(companies_df[\"company\"], num_edges)\n",
+    "dest_companies = np.random.choice(companies_df[\"company\"], num_edges)\n",
+    "\n",
+    "# Generate random timestamps between 2010 and 2025\n",
+    "start_date = pd.to_datetime(\"2010-01-01\")\n",
+    "end_date = pd.to_datetime(\"2025-12-31\")\n",
+    "time_range = end_date - start_date\n",
+    "random_seconds = np.random.rand(num_edges) * time_range.total_seconds()\n",
+    "random_timestamps = start_date + pd.to_timedelta(random_seconds, unit=\"s\")\n",
+    "\n",
+    "# Create a list of possible relationship types\n",
+    "relationship_types = [\"acquisition\", \"partnership\", \"investment\", \"merger\", \"collaboration\"] # Add more types as needed\n",
+    "\n",
+    "# Assign random relationship types\n",
+    "random_types = np.random.choice(relationship_types, num_edges)\n",
+    "\n",
+    "# Create the edges_df DataFrame\n",
+    "edges_df = pd.DataFrame({\n",
+    "    \"src\": src_companies,\n",
+    "    \"dest\": dest_companies,\n",
+    "    \"timestamp\": random_timestamps,\n",
+    "    \"type\": random_types\n",
+    "})\n",
+    "\n",
+    "display(edges_df.head(3))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "rufg0pmd3ga",
+   "metadata": {},
+   "source": [
+    "## Standard Graph Visualization\n",
+    "\n",
+    "First, let's create a standard Graphistry visualization using the default maps parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c25b004f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g = graphistry.bind(source=\"src\", destination=\"dest\", node=\"company\") \\\n",
+    "    .nodes(companies_df) \\\n",
+    "    .edges(edges_df) \\\n",
+    "    .layout_settings(play=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71e9d1ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g.plot()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4db77e5d",
+   "metadata": {},
+   "source": [
+    "## Configure: Custom datasets and layers\n",
+    "\n",
+    "* Customize the data available to Kepler"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41441609",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create Kepler encoding with datasets and layers\n",
+    "g2 = g \\\n",
```

Comment:
Is there a 'minimal' simple map we can get generated 'for free' on this data ? requiring all this config for a maps hello world on it looks quite complicated...

Reply:
addressed in slack